<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<<<<<<< HEAD
<testsuite skipped="0" hostname="ED-MacBook-Pro.local" name="net.ed.LoginServletTest" tests="1" failures="0" timestamp="01 Mar 2018 17:00:50 GMT" time="0.037" errors="0">
  <testcase name="doGetTest" time="0.037" classname="net.ed.LoginServletTest"/>
=======
<testsuite skipped="0" hostname="ED-MacBook-Pro.local" name="net.ed.LoginServletTest" tests="3" failures="1" timestamp="01 Mar 2018 17:38:32 GMT" time="26.816" errors="0">
  <testcase name="getURLTest" time="2.566" classname="net.ed.LoginServletTest"/>
  <testcase name="enterUserNameTest" time="10.330" classname="net.ed.LoginServletTest"/>
  <testcase name="enterUserPasswordTest" time="13.920" classname="net.ed.LoginServletTest">
    <failure type="java.lang.AssertionError" message="expected [Yahoo] but found [Yahoo -Â login]">
      <![CDATA[java.lang.AssertionError: expected [Yahoo] but found [Yahoo -Â login]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at net.ed.LoginServletTest.enterUserPasswordTest(LoginServletTest.java:87)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- enterUserPasswordTest -->
>>>>>>> dev
=======
<testsuite skipped="0" hostname="ED-MacBook-Pro.local" name="net.ed.LoginServletTest" tests="4" failures="0" timestamp="01 Mar 2018 18:26:37 GMT" time="11.338" errors="0">
  <testcase name="getURL_Test" time="2.633" classname="net.ed.LoginServletTest"/>
  <testcase name="enterUserName_Test" time="1.317" classname="net.ed.LoginServletTest"/>
  <testcase name="enterPassword_Test" time="7.333" classname="net.ed.LoginServletTest"/>
  <testcase name="getCookies_Test" time="0.055" classname="net.ed.LoginServletTest"/>
>>>>>>> dev
</testsuite> <!-- net.ed.LoginServletTest -->
